### PR TITLE
Bumping example's React version to 0.13.x, and wrapping the 'window' …

### PR DESCRIPTION
**Pull request by Velveeta**
…injection in a typeof check so we can safely import this module within Node.js for server-side rendering.

### CODE REVIEW

File: example/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "homepage": "https://github.com/chenglou/react-spinner",
   "dependencies": {
-    "react": "^0.12.1"
+    "react": "^0.13.0"
```
Comment (chenglou) [inline example/package.json]: Actually, can you use `>=0.13.0`?

Comment (Velveeta) [inline example/package.json]: Good call, I'll make that change right now.

**Comment by Velveeta** [comment]
Is there anything else needed for this?

**Comment by chenglou** [comment]
Nope, sorry I totally forgot about this.

**Comment by chenglou** [comment]
@Velveeta pushed as 0.2.3. Tell me if anything goes wrong!

**Comment by Velveeta** [comment]
Beautiful, thank you very much :)

**Pull request merged**